src/main/java/uk/gov/cslearning/record/repository/InviteRepository.java

```diff
@@ -1,15 +1,16 @@
 package uk.gov.cslearning.record.repository;
 
-import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 import uk.gov.cslearning.record.domain.Invite;
 
 import java.util.Collection;
+import java.util.Optional;
 
 @Repository
 public interface InviteRepository extends CrudRepository<Invite, Integer> {
 
-    @Query("SELECT i FROM Invite i INNER JOIN Event e ON i.event = e WHERE e.uid = ?1")
```

Comment:
You don't need to explicity mention the inner join here. You could write this as...
```
@Query("SELECT i FROM Invite i WHERE i.event.uid = :eventUid")
Collection<Invite> findByEventId(@Param("eventUid") String eventUid);
```